src/core/types.ts

```diff
@@ -160,11 +160,7 @@ export interface QueryObserverOptions<
    * When set to `['data', 'error']`, the component will only re-render when the `data` or `error` properties change.
    * When set to `tracked`, access to properties will be tracked, and the component will only re-render when one of the tracked properties change.
```

Comment:
the comment is no longer accurate here :)

Reply:
Yah - this is for sure part of the docs i need to update